ui/app/AppLayouts/Profile/Sections/Ens/Search.qml

```diff
@@ -48,7 +48,8 @@ Item {
 
     ModalPopup {
         id: transactionDialog
-        title: qsTr("TODO: replace this for the transaction dialog")
+        //% "TODO: replace this for the transaction dialog"
+        title: qsTrId("todo--replace-this-for-the-transaction-dialog")
```

Comment:
lol, this one didn't need to use `qsTr` but oh well, it's not a big deal. QT will cull that entry once we remove it